directus-aws/index.md

```diff
@@ -0,0 +1,206 @@
+
+Click on the `Launch Instance` button, and this will create a new EC2 instance for you
+
+### Set up permission
+
+To ensure that the EC2 instance can be accessed and operated from anywhere, head over to the security group in your EC2 instance and add the following inbound rules:
+
+| Name  | Security group rule ID | IP version | Type | Protocol | Port Range | Source |
+| --- | --- | --- | --- | --- | --- | --- |
+| -  | your default security group | IPv4 | SSH | TCP | 22 | 0.0.0.0/0 |
+| -  | your default security group | IPv4 | HTTP | TCP | 80 | 0.0.0.0/0 |
+| -  | your default security group | IPv4 | HTTPS | TCP | 443 | 0.0.0.0/0 |
+
+![EC2 Network security settings inbound rules](inbound.png)
+
+Also, create an outbound rule for connecting to the database:
+
+| Name  | Security group rule ID | IP version | Type | Protocol | Port Range | Source |
+| --- | --- | --- | --- | --- | --- | --- |
+| -  | your default security group | IPv4 | PostgreSQL | TCP | 5432 | 0.0.0.0/0 |
+
+![EC2 Network security settings outbound rules](outbound.png)
+Save these rules.
+
+## Set up Directus on AWS EC2
+
+On the EC2 instance page, click on the connect button to connect to your AWS EC2 instance using EC2 Instance Connect (you can also connect to EC2 via other methods such as a session manager or SSH client).
+
+![EC2 instance connect page](ec2-connect.png)
+
+Once connected, run the command to install Docker on your EC2 server:
+
+```bash
+sudo yum install -y docker
+```
+
+Next, install Docker-compose with the command:
+
+```bash
+sudo curl -L https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m) -o /usr/local/bin/docker-compose
+```
+
+Grant permission to run the `docker-compose` command without using the `sudo` command
+
+```bash
+sudo chmod +x /usr/local/bin/docker-compose
+```
+
+Start the docker service with the command:
+
+```bash
+sudo service docker start
+```
+
+Create a new directory called `directus` and navigate into the directory with the command:
+
+```bash
+mkdir directus && cd directus
+```
+
+In the `directus` directory, create a `docker-compose.yml` file and update its content using `vim` with the command:
+
+```bash
+touch docker-compose.yml && vim docker-compose.yml
+```
+
+This will open the `docker-compose.yml` file in `vim`, to edit the file, press `i` and paste the following Directus configuration:
+
+```yml
+version: "3"
+services:
+  directus:
+    image: directus/directus:10.8.3
+    ports:
+      - 80:80
+    volumes:
+      - ./uploads:/directus/uploads
+      - ./extensions:/directus/extensions
+    environment:
+      PORT: 80
+      KEY: "replace-with-random-value"
+      SECRET: "replace-with-random-value"
+      ADMIN_EMAIL: "admin@example.com"
+      ADMIN_PASSWORD: "d1r3ctu5"
+      DB_CLIENT: "pg"
+      DB_HOST: "YOUR_RDS_DB_URL"
+      DB_PORT: 5432
+      DB_DATABASE: "postgres"
+      DB_USER: "YOUR_DB_USER"
+      DB_PASSWORD: "YOUR_DB_PASSWORD"
```

Comment:
You missed the part where you explain where these values can be found. And the DB_HOST